### DISCUSSION TITLE
Fastfetch is showing wrong linux distro

### DISCUSSION THREAD

**Discussion by FavianFrancis** [fastfetch-cli/fastfetch#1434] [Q&A] [answered]
I'm using Linux Lite but Fastfetch is showing the OS as Xubuntu. How can I fix this? Thank you.

**Comment by CarterLi** [comment]
Can be a fastfetch bug. Please upload the content of `/etc/os-release` and output of `fastfetch -s os --format json`

**Reply by FavianFrancis** [reply]
![os-release](https://github.com/user-attachments/assets/10bdbdbf-6e28-4dab-82a4-50585f1b42fd)
![fastfetch -s os](https://github.com/user-attachments/assets/14b55f6c-985b-4650-be0b-8668f341055b)

**Reply by CarterLi** [reply, marked as answer]
Ok. Will be fixed in the next version

**Reply by FavianFrancis** [reply]
Thank you 🙏

**Comment by CarterLi** [comment]
Can you test this? https://github.com/fastfetch-cli/fastfetch/actions/runs/12272658289

**Reply by FavianFrancis** [reply]
It works, thank you for fixing it 🙏

![Screenshot_2024-12-11_09-06-06](https://github.com/user-attachments/assets/c7ec3f52-542e-4e6b-a7dc-4a220e9a8216)
